internal/function/blame.go

```diff
@@ -145,7 +153,7 @@ func (b *Blame) Eval(ctx *sql.Context, row sql.Row) (interface{}, error) {
 		return nil, err
 	}
 
-	bg, err := NewBlameGenerator(commit, fIter)
+	bg, err := NewBlameGenerator(commit, fIter, ctx)
```

Comment:
Context is by convention always the first parameter if a function takes a context.

Reply:
Fixed, thanks, learnt something new.